perseo-main/pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
```

Comment:
Looking to reference PR https://github.com/telefonicaid/perseo-core/pull/180, CNR should be flushed in this PR.

Reply:
Done in https://github.com/telefonicaid/perseo-core/pull/198/commits/5b7129e164ad0e83d9e1154d61acbb5277d3021e